scripts/install.sh

```diff
@@ -35,12 +41,7 @@ kubectl apply -f platform/flux/secrets/secret-flux-key.yaml
 kubectl apply -f platform/flux/flux.yaml
 
 # Wait till all pods against flux deployment are deployed & then print flux public key
-kubectl -n flux wait --timeout=200s --for condition=ready pod -l release=stakater-infra-flux
+kubectl -n flux wait --timeout=200s --for condition=Ready pod -l release=stakater-infra-flux
 echo -e "\n======== Add the following Flux Public Key to your git repository ========"
 #kubectl -n flux logs deployment/flux | grep identity.pub | cut -d '"' -f2
-cat ./configs/flux.pub
-
-# Wait for dashboard to be ready & then print dashboard access token
```

Comment:
move it back here, install dashboard from the script and wait till its pods are ready and then print token !